### ISSUE TITLE
django-inlinecss-redux now points here

### ISSUE THREAD

**Issue by WhyNotHugo** [roverdotcom/django-inlinecss#52]
Mostly FYI, feel free to close this after reading it :)

In 2017 I needed a small bug fixed, but this project wasn't active. At the time, I made a fork (`django-inlinecss-redux`) and pushed that to PyPI.

I see now that this came back to life at some point, and my issue was fixed here too. I've therefore archived `django-inlinecss-redux`, and added [a big sign](https://github.com/WhyNotHugo/django-inlinecss-redux#django-inlinecss-redux) pointing people here (I don't think anyone else used it anyway).

If you ever do need a hand with maintenance or addressing the smaller issue, do let me know, happy to help. I'm noticing a few issues still open which have been addresses already and are safe to close.

Cheers, and thanks for maintaining this.

**Comment by huygens545** [comment]
> I see now that this came back to life at some point, and my issue was fixed here too
Good to know! Learning how to send mail, I've searched just now a package doing that and that's the only one fulfilling this niche apparently!
Hope it'll get more love!

**Comment by WhyNotHugo** [comment]
Is there any breakage on recent Python / Django versions?

If so, I can bring back django-inlinecss-redux (assuming you have fixes for those issues).

**Comment by Mogost** [comment]
Yes. There are some problems with the new django https://github.com/roverdotcom/django-inlinecss/pull/58

**Comment by sshishov** [comment]
@WhyNotHugo could you ask to be a maintainer of this project? As we are using this project and cannot use it now for Django4
